sklearn/linear_model/tests/test_base.py

```diff
@@ -105,12 +106,14 @@ def test_raises_value_error_if_positive_and_sparse():
         reg.fit(X, y)
 
 
-def test_raises_value_error_if_sample_weights_greater_than_1d():
+def test_raises_value_error_if_sample_weights_greater_than_1d(global_random_seed):
     # Sample weights must be either scalar or 1D
 
     n_sampless = [2, 3]
     n_featuress = [3, 2]
 
+    rng = np.random.RandomState(global_random_seed)
```

Comment:
Test passes with all seeds. I am not sure if it is strictly necessary to test this for more than one seed though. We are testing different random combinations of `X, y` and `sample_weights`